### DISCUSSION TITLE
I can't renew my student status, got rejected in a minute

### DISCUSSION THREAD

**Discussion by alperenorence** [community/community#57775] [GitHub Education]
### Hi everyone,

Hello. I'm 17 years old high school student in Turkey. I am using GitHub for a couple of years. Before this year, I was accepted to Student Pack twice. I tried to renew my subscription, but I guess it stuck to the AI filter.

In Turkey, we can get a student document about the current status of education for that people. This document is issued by the education ministry and given from the government's website. (turkiye.gov.tr) This document clearly shows the name, surname, full name of the school and given date of the document. This document is Turkish and also verifiable via a .gov website.

Almost all of the high schools in Turkey don't provide a .edu mail address to their students. Because of that, I don't have a .edu mail and the only way to provide my education status is this document. I accepted to Student Pack twice via this document. But now, I tried to apply multiple times, and all of them are rejected in a couple of seconds.

I printed the original document and its English translation (used DeepL) and I put them on the floor side by side. I take a photo via my phone. This photo contains all the necessary information. I sent that image, but it got rejected in a minute. And I got only these two suggestions from GitHub. The first one is image is from a third party, use an official image. But this image is already original. And the second one is to take it via webcam. Since I don't have a webcam, I connected my phone as a webcam (Continuity Camera on iPhone) and took a photo of the documents in the same way. But this way, all of the images were blurry. I submitted these but failed. I got warnings about it doesn't contain my name or the school name. It was right, the texts are not readable since it was blurry.

I tried all ways, including taking these photos from different angles. But all got failed. I tried on different days but it not worked. Also, I was going to create a support ticket but GitHub doesn't allow a support ticket for rejection to Student Pack. I am a REAL student, I can prove it to a person but this new GitHub AI filter blocks me. I need help.

I'm adding three images. The first one shows given suggestions when using my phone. The second one shows given suggestions when using a webcam. And third one shows my previous approvals.

![Image taken from phone](https://github.com/community/community/assets/46728961/09042849-75a5-405a-8775-35fcf4bfb11c)
![Image taken from webcam](https://github.com/community/community/assets/46728961/6c918b4f-2b6d-4fb4-a081-22f1c8b4dab9)
![Previus approvals](https://github.com/community/community/assets/46728961/ad95e11f-1823-4119-8f45-e5030e00a4e9)

**Comment by mrepol742** [comment]
Github Education now requires school issued email address and academic documentation in papers(theyre not accepting screenshots or online documents to prevent forgery). I also faced this but no matter what i do they keep rejecting it. At this stage there is nothing to do unless they changed or update there system.

**Comment by dangtiendung1201** [comment]
I have the same problem. I have just received an email requiring reverification. I used my students ID which was accepted in April 2023 and then got rejected by AI filter. Moreover, I cannot use my back camera to capture the proof.
